modules/machine-health-checks-about.adoc

```diff
@@ -6,32 +6,29 @@
 [id="machine-health-checks-about_{context}"]
 = About machine health checks
 
-You can define conditions under which machines in a cluster are considered unhealthy by using a `MachineHealthCheck` resource.
-Machines matching the conditions are automatically remediated.
+Machine health checks automatically repair unhealthy machines in a particular machine pool.
```

Comment:
This set of changes does not have to do with the current PR

Reply:
I'm sorry about this, it is a rogue commit from another PR. I'm working on removing it now. Thanks and apologies!